vendor/github.com/bitrise-steplib/bitrise-step-build-router-start/bitrise/bitrise.go

```diff
@@ -355,6 +359,58 @@ func (artifact BuildArtifact) DownloadArtifact(filepath string) error {
 	return err
 }
 
+// AbortBuild ...
+func (app App) AbortBuild(buildSlug string, abortReason string) error {
+	params := make(map[string]interface{})
+	params["abort_reason"] = abortReason
+	params["abort_with_success"] = false
+	params["skip_notifications"] = true
+
+	b, err := json.Marshal(params)
+	if err != nil {
+		return nil
+	}
+	rm := abortRequest{BuildAbortParams: b}
```

Comment:
Is this correct?

Reply:
Abort API shows
![image](https://user-images.githubusercontent.com/60897705/89251436-dbbe9880-d5cb-11ea-81fc-ca7ea4caff5e.png)